### DISCUSSION TITLE
feature: octo.nvim ui for gh

### DISCUSSION THREAD

**Discussion by DrummyFloyd** [LazyVim/LazyVim#3478] [Extra Requests]
### Did you check the docs?

- [X] I have read all the LazyVim docs

### Is your feature request related to a problem? Please describe.

no

### Describe the solution you'd like

integrate https://github.com/pwntester/octo.nvim
to enforce the work inside terminal 

### Describe alternatives you've considered

dunno if it fit your need too , if not will add by my own , bbut i think it's a good plugin to have , especially if you manage several gh repo

### Additional context

.

**Comment by DrummyFloyd** [comment]
for ppl who work also with gitlab
there is also one for gitlab => https://github.com/harrisoncramer/gitlab.nvim

**Comment by DrummyFloyd** [comment]
@folke  thank for the Extras, 

question about how you chose your keymaps
=> how do you chose these keymap ? 

because i want to add also `Gitlab.nvim `as plugins on my config, but dunno how to deal with both with keymaps
how will you achieve something similar ? =)

**Comment by muneebusmani** [comment]
bruh if anyone knows then tell me how do we create a popup window, for scripts like ng generate, dotnet-tool-aspnet-codegenerator , and like laravel artisan, basically just a selection menu inside neovim
